src/main/java/com/minecolonies/coremod/colony/buildings/workerbuildings/BuildingMiner.java

```diff
@@ -438,7 +438,7 @@ private void loadLadderPos()
         final Set<BlockPos> ladderPos = map.getOrDefault("ladder", new HashSet<>());
         if (cobblePos.isEmpty() || ladderPos.isEmpty())
         {
-            updateTEDataFromSchematic();
+            saveUpdateTEDataFromSchematic();
```

Comment:
just noticing now, but incase we got a bad schematic with missing data this would reload-spam all day long

Reply:
Will make it a seperate PR after lunch